backend/pyproject.toml

```diff
@@ -58,3 +58,15 @@ omit = [
     "wsgi.py",
     "manage.py"
     ]
+
+[tool.mypy]
+plugins = [
+    mypy_django_plugin.main,
+    mypy_drf_plugin.main
+]
+ignore_missing_imports = True
+strict = True
+
+[[tool.mypy.overrides]]
+module = 'django-stubs'
+django_settings_module = 'backend.settings'
```

Comment:
You need to use quotes and general we use double quotes inside the pyproject.toml. 

```toml
plugins = [
    "mypy_django_plugin.main",
    "mypy_drf_plugin.main"
]
```

`True` needs to be lower case inside pyproject.toml. Will indicate an error when you run mypy 

`pyproject.toml: Invalid value`

for  django-stubs have a look at the github repo, they have an example for the [pyproject.toml](https://github.com/typeddjango/django-stubs)